### PR TITLE
LPS-87269 Rename portletPreferences to preferences to attain coherence with settings parameter

### PR DESCRIPTION
**Pull request by izaera**

### CODE REVIEW

**Comment by izaera** [comment]
ci:test

File: modules/apps/frontend-js-portlet-extender/frontend-js-portlet-extender/src/main/java/com/liferay/frontend/js/portlet/extender/internal/portlet/JSPortlet.java [hunk 1]
```diff
@@ -63,26 +64,27 @@ public void render(
...
+				new String[] {portletElementId});
+
+			printWriter.print(html);
+
+			String javascript = StringUtil.replace(
+				_TPL_JAVA_SCRIPT,
+				new String[] {
+					"[$CONTEXT_PATH$]", "[$PACKAGE_NAME$]",
+					"[$PACKAGE_VERSION$]", "[$PORTLET_ELEMENT_ID$]",
+					"[$PORTLET_NAMESPACE$]", "[$PREFERENCES$]", "[$SETTINGS$]"
```
Comment (jbalsas) [inline modules/apps/frontend-js-portlet-extender/frontend-js-portlet-extender/src/main/java/com/liferay/frontend/js/portlet/extender/internal/portlet/JSPortlet.java]: Shouldn't this be `PORTLET_PREFERENCES`?

File: modules/apps/frontend-js-portlet-extender/frontend-js-portlet-extender/src/main/java/com/liferay/frontend/js/portlet/extender/internal/portlet/JSPortlet.java [hunk 2]
```diff
@@ -63,26 +64,27 @@ public void render(
...
+				},
+				new String[] {
+					renderRequest.getContextPath(), _packageName,
+					_packageVersion, portletElementId,
+					renderResponse.getNamespace(),
+					_getPreferences(renderRequest), _getSettings()
```
Comment (jbalsas) [inline modules/apps/frontend-js-portlet-extender/frontend-js-portlet-extender/src/main/java/com/liferay/frontend/js/portlet/extender/internal/portlet/JSPortlet.java]: `_getPortletPreferences`?

File: modules/apps/frontend-js-portlet-extender/frontend-js-portlet-extender/src/main/resources/com/liferay/frontend/js/portlet/extender/internal/portlet/dependencies/bootstrap.js.tpl [hunk 3]
```diff
@@ -12,13 +12,17 @@
 			}
 
 			if (initializer) {
+				var preferences = JSON.parse('[$PREFERENCES$]');
```
Comment (jbalsas) [inline modules/apps/frontend-js-portlet-extender/frontend-js-portlet-extender/src/main/resources/com/liferay/frontend/js/portlet/extender/internal/portlet/dependencies/bootstrap.js.tpl]: Based on https://github.com/brianchandotcom/liferay-portal/pull/67948#issuecomment-463342460, I'm not sure we'll get away with this... 🤔

**Comment by izaera** [comment]
OK. I will resend renaming all but the file name. Given that Brian asked "if he could rename the file", I assume there will be no problem in just leaving the file with a different name.

**Comment by jbalsas** [comment]
Yeah, I assume so too 👍

**Comment by izaera** [comment]
Sent as https://github.com/brianchandotcom/liferay-portal/pull/68193

**Pull request closed without merging**